CONTRIBUTING.md

```diff
@@ -11,3 +11,7 @@ with how you'd like to contribute, if you want to introduce yourself.
 
 Feature planning is done on a Trello board; please open an issue and mention
 [@cheshire137](https://github.com/cheshire137) to get added.
+
+Please know we have a [Code of
+Conduct](https://github.com/cheshire137/overwatch-team-comps/blob/master/CODE_OF_CONDUCT.md) in place.
```

Comment:
@cheshire137 oh cool, thanks! I've added this in https://github.com/cheshire137/overwatch-team-comps/pull/23/commits/1bd44a1b7401223118fd47aca05f5f7f2c88bb94